content/ui-patterns/feature-onboarding.mdx

```diff
@@ -133,7 +133,7 @@ Treat feature onboarding as a story or a journey with a beginning, middle, and e
 
 A teaching bubble is a [popover](https://primer.style/css/components/popover) that calls attention to a feature in a specific part of the page. Generally, teaching bubbles should be used to educate the user and enrich the task at hand.
 
-### Usage guidelines
+### Usage guidelines for teaching bubble
```

Comment:
This one was to address: [no-duplicate-heading](https://github.com/updownpress/markdown-lint/blob/master/rules/024-no-duplicate-header.md).

The rationale is:

> Some markdown parses generate anchors for headers based on the header name, and having headers with the same content can cause problems with this.

which was new to me! Sure enough, we have this issue with [usage guidelines](https://primer.style/design/ui-patterns/feature-onboarding#usage-guidelines.